make-cookbook.lisp

```diff
@@ -48,7 +51,7 @@
    "web.md"
    "web-scraping.md"
    "websockets.md"
-   ;; "misc.md"
+   "misc.md"
```

Comment:
In there, there are two tips. One about adjust-array: if it's relevant why not put it to the appropriate chapter. Is this little page really relevant? I'm not confident about it, I'd leave it outside the epub.